### PR TITLE
Added years filter

### PR DESCRIPTION
**Pull request by jwest60**
image

image

The list of years also updates dynamically depending on how much data is available; if you filter and only measurements from 2018-2020 exist, then any other years will not be available to sort by.

image

### CODE REVIEW

**Comment by jwest60** [comment]
#241

**Comment by jwest60** [comment]
Checkboxes now appear in a horizontal list instead of a vertical one

Also applied the rest of the changes from #241 since they weren't added to master

**Comment by jwest60** [comment]
#254

File: resources/js/screens/Statistics.vue [hunk 1]
```diff
@@ -526,6 +565,15 @@
...
         this.$refs.state.clear()
       },
+
+      filterYear(year) {
+        if (this.yearsFilter.includes(year)) {
+          this.yearsFilter = this.yearsFilter.filter(y => y !== year)
+        } else {
+          this.yearsFilter.push(year)
+        }
+        console.log(this.yearsFilter)
```
Comment (almasaeed2010) [inline resources/js/screens/Statistics.vue:575]:
```suggestion
```

**Pull request merged**